client/state/notifications/index.js

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { NOTIFICATIONS_UNSEEN_COUNT_SET } from 'state/action-types';
+
+export const unseenCount = ( state = 0, action ) =>
```

Comment:
For default state, do we need to differentiate between a "loading" state, vs when folks actually have an unread count of zero? If we need the former, consider making default state something like `null` so you can tell between the two states.

This is a pretty trivial reducer/selector, but do y'all mind adding a quick unit test?

Reply:
Most of the time, it makes sense distinguishing between
- "unknown" state, represented by `null`
- "known to be zero" represented by `0`